slides/en/net/conversion/pptx-to-word/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title:  Convert PPTX to Word in C#
+url: /net/conversion/pptx-to-word/
+keywords: Convert PPTX to Word, PPTX to Word, PPTX to DOC, PowerPoint to Word, C# API, .NET Library
+description: Convert PPTX to Word in C#. Use .NET library API to convert PowerPoint to Word
+---
+
+{{< blocks/products/pf/main-wrap-class isAutogenPage="true" >}}
+{{< blocks/products/pf/feature-page-wrap >}}
+
+{{< blocks/products/pf/feature-page-header h1="Convert PPTX to Word in C#" h2="Powerful cross-platform .NET API for converting PowerPoint to Word using C# code on NET Framework, .NET Core, Windows Azure, Mono or Xamarin Platforms" >}}
+
+{{% blocks/products/pf/feature-page-section h2="Convert PowerPoint to Word using Aspose.Slides and Aspose.Words" %}}
+
+[**Aspose.Slides for .NET**](https://products.aspose.com/slides/net/) and [**Aspose.Words for .NET**](https://products.aspose.com/words/net/) are powerful .NET libraries used to manipulate and convert PowerPoint presentations, Word documents, and other files. When you convert PowerPoint to Word, you are essentially moving the contents of a presentation's slides to pages in a Word document.
+
+{{% /blocks/products/pf/feature-page-section %}}
+
+
+
+
+{{% blocks/products/pf/feature-page-section  h2="Convert PowerPoint to Word in C#" %}}
+You can convert PPTX to Word quickly with just a few lines of code
+
+{{% blocks/products/pf/agp/code-block title="C# code for converting PowerPoint to Word" offSpacer="true" %}}
+```cs
+using var presentation = new Presentation();
+var doc = new Document();
+var builder = new DocumentBuilder(doc);
+foreach (var slide in presentation.Slides)
+{
+    // generates and inserts slide image
+    using var bitmap = slide.GetThumbnail(1, 1);
+    using var stream = new MemoryStream();
+    bitmap.Save(stream, ImageFormat.Png);
+    stream.Seek(0, SeekOrigin.Begin);
+    using var skBitmap = SKBitmap.Decode(stream);
+    builder.InsertImage(skBitmap);
+
+    // inserts slide texts
+    foreach (var shape in slide.Shapes)
+    {
+        if (shape is AutoShape autoShape)
+        {
+            builder.Writeln(autoShape.TextFrame.Text);
+        }
+    }
+
+    builder.InsertBreak(BreakType.PageBreak);
+}
+```
```

Comment:
@EmSeriki you missed the code for saving the word document:
```doc.Save("document.docx");```

Reply:
I took the code from the PowerPoint to Word article in our documentation. I'll make corrections there too. Thank you.